### DISCUSSION TITLE
PCAP Steno retention

### DISCUSSION THREAD

**Discussion by DashCarlos** [Security-Onion-Solutions/securityonion#2110] [Unsupported Versions] [answered]
Is there any parameter to determine the number of days I want to retain the pcap files?
It filled 60GB on my system ...

> {
  "Threads": [
    { "PacketsDirectory": "/nsm/pcap"
    , "IndexDirectory": "/nsm/pcapindex"
    , "MaxDirectoryFiles": 30000
    , "DiskFreePercentage": 10
    }
  ]
  , "StenotypePath": "/usr/bin/stenotype"
  , "Interface": "bond0"
  , "Port": 1234
  , "Host": "127.0.0.1"
  , "Flags": ["-v", "--uid=stenographer", "--gid=stenographer"]
  , "CertPath": "/etc/stenographer/certs"
}

**Comment by dougburks** [comment, marked as answer]
Please see Discussion https://github.com/Security-Onion-Solutions/securityonion/discussions/2055 and Issue https://github.com/Security-Onion-Solutions/securityonion/issues/2095.
